tiatoolbox/utils/visualization.py

```diff
@@ -33,6 +34,19 @@ def random_colors(num_colors, bright=True):
     return colors
 
 
+def colourise_image(img, cmap="viridis"):
+    """If input img is single channel, colourise it
+    using specified colourmap.
+    """
```

Comment:
Just minor formatting. The first line of the docstring should be a single line with a full stop.

Additional detail is then in a paragraph after a blank line. Also missing args and returns here.